### ISSUE TITLE
[vala] Failed to build

### ISSUE THREAD

**Issue by uberhacker** [skycocker/chromebrew#4474]
```
/usr/local/lib/gcc/armv7l-cros-linux-gnueabihf/8.3.0/../../../../armv7l-cros-linux-gnueabihf/bin/ld: /usr/local/lib/gcc/armv7l-cros-linux-gnueabihf/8.3.0/../../../libgvc.so: undefined reference to `sincos'
/usr/local/lib/gcc/armv7l-cros-linux-gnueabihf/8.3.0/../../../../armv7l-cros-linux-gnueabihf/bin/ld: /usr/local/lib/gcc/armv7l-cros-linux-gnueabihf/8.3.0/../../../libgvc.so: undefined reference to `sqrt'
/usr/local/lib/gcc/armv7l-cros-linux-gnueabihf/8.3.0/../../../../armv7l-cros-linux-gnueabihf/bin/ld: /usr/local/lib/gcc/armv7l-cros-linux-gnueabihf/8.3.0/../../../libgvc.so: undefined reference to `floor'
/usr/local/lib/gcc/armv7l-cros-linux-gnueabihf/8.3.0/../../../../armv7l-cros-linux-gnueabihf/bin/ld: /usr/local/lib/gcc/armv7l-cros-linux-gnueabihf/8.3.0/../../../libgvc.so: undefined reference to `tan'
/usr/local/lib/gcc/armv7l-cros-linux-gnueabihf/8.3.0/../../../../armv7l-cros-linux-gnueabihf/bin/ld: /usr/local/lib/gcc/armv7l-cros-linux-gnueabihf/8.3.0/../../../libgvc.so: undefined reference to `acos'
/usr/local/lib/gcc/armv7l-cros-linux-gnueabihf/8.3.0/../../../../armv7l-cros-linux-gnueabihf/bin/ld: /usr/local/lib/gcc/armv7l-cros-linux-gnueabihf/8.3.0/../../../libgvc.so: undefined reference to `exp'
/usr/local/lib/gcc/armv7l-cros-linux-gnueabihf/8.3.0/../../../../armv7l-cros-linux-gnueabihf/bin/ld: /usr/local/lib/gcc/armv7l-cros-linux-gnueabihf/8.3.0/../../../libgvc.so: undefined reference to `sin'
/usr/local/lib/gcc/armv7l-cros-linux-gnueabihf/8.3.0/../../../../armv7l-cros-linux-gnueabihf/bin/ld: /usr/local/lib/gcc/armv7l-cros-linux-gnueabihf/8.3.0/../../../libgvc.so: undefined reference to `hypot'
/usr/local/lib/gcc/armv7l-cros-linux-gnueabihf/8.3.0/../../../../armv7l-cros-linux-gnueabihf/bin/ld: /usr/local/lib/libpathplan.so.4: undefined reference to `pow'
/usr/local/lib/gcc/armv7l-cros-linux-gnueabihf/8.3.0/../../../../armv7l-cros-linux-gnueabihf/bin/ld: /usr/local/lib/gcc/armv7l-cros-linux-gnueabihf/8.3.0/../../../libgvc.so: undefined reference to `atan2'
/usr/local/lib/gcc/armv7l-cros-linux-gnueabihf/8.3.0/../../../../armv7l-cros-linux-gnueabihf/bin/ld: /usr/local/lib/gcc/armv7l-cros-linux-gnueabihf/8.3.0/../../../libgvc.so: undefined reference to `cos'
collect2: error: ld returned 1 exit status
make[4]: *** [Makefile:529: valadoc] Error 1
make[4]: Leaving directory '/usr/local/tmp/crew/vala-0.50.1.tar.xz.dir/vala-0.50.1/valadoc'
make[3]: *** [Makefile:586: all-recursive] Error 1
make[3]: Leaving directory '/usr/local/tmp/crew/vala-0.50.1.tar.xz.dir/vala-0.50.1/valadoc'
make[2]: *** [Makefile:443: all] Error 2
make[2]: Leaving directory '/usr/local/tmp/crew/vala-0.50.1.tar.xz.dir/vala-0.50.1/valadoc'
make[1]: *** [Makefile:544: all-recursive] Error 1
make[1]: Leaving directory '/usr/local/tmp/crew/vala-0.50.1.tar.xz.dir/vala-0.50.1'
make: *** [Makefile:432: all] Error 2
vala failed to install: `make V=0 -j4` exited with 2
```

**Comment by uberhacker** [comment]
For x86_64:
```
checking for libgvc >= 2.16... no
configure: error: Package requirements (libgvc >= 2.16) were not met:

No package 'libgvc' found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables LIBGVC_CFLAGS
and LIBGVC_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.
vala failed to build: `./configure --prefix=/usr/local --libdir=/usr/local/lib64` exited with 1
```

**Comment by uberhacker** [comment]
Fixed in #4485.
